### PR TITLE
checks whether the failure callback has a error object

### PR DESCRIPTION
**Pull request by mordrax**
Relevant flow discussion:
https://www.flowdock.com/app/blake/help/threads/bDXqr0SyE9TzKe8zUOJ7ELkWTEA

### CODE REVIEW

File: lib/hopscotch/runners/default.rb [hunk 1]
```diff
@@ -21,6 +21,10 @@ def call(fn, failure:, success:)
         error = nil
         result = nil
 
+        if failure.arity != 1
+          raise ArgumentError, "failure lambda requires a error value argument."
```
Comment (lachie) [inline lib/hopscotch/runners/default.rb]: *an

**Review by lachie — COMMENTED** [review]
This is fine, but we should hold off merging til we're ready to fix the bugs that come up.

**Comment by kunzmann** [comment]
So bump the version in a separate PR just before releasing to rubygems? Or should that happen here?

**Comment by mordrax** [comment]
So I'm closing this due to spring cleaning and as it doesn't seem like it'll be merged any time soon.
However, it does seem useful to know when a failure has occurred, I hope this is eventually picked up.

**Pull request closed without merging**